allensdk/brain_observatory/behavior/image_api.py

```diff
@@ -24,6 +24,8 @@ def __eq__(self, other):
         c = self.unit == other.unit
         return a and b and c
 
+    def __array__(self):
+        return self.data
```

Comment:
since self.data is not runtime-requried to be a numpy array, it would be safer if this method called np.array()

Reply:
Resolved by bda1d6c684c7befe7efb61ba3972cb8a1a3c99c9